docs_user/modules/con_openshift-preparation-for-block-storage-adoption.adoc

```diff
@@ -111,37 +84,28 @@ spec:
         name: iscsid.service
 ----
 +
-If you are using labels to restrict the nodes where the Block Storage services are running you need to use a `MachineConfigPool` as described in
-the xref:about-node-selector_planning[About node selector] to limit the effects of the
-`MachineConfig` to only the nodes where your services may run.
+If you use labels to restrict the nodes where the Block Storage services run, you must use a `MachineConfigPool` to limit the effects of the
+`MachineConfig` to only the nodes where your services might run. For more information, xref:about-node-selector_planning[About node selector].
 +
 If you are using a single node deployment to test the process, replace `worker` with `master` in the `MachineConfig`.
-
-//For production deployments using iSCSI volumes, we always recommend setting up
-//multipathing, please look at the <<multipathing,multipathing section>> to see
-//how to configure it. kgilliga: Commented out because multipathing module doesn't exist yet. Update with xref for beta.
++
+For production deployments that use iSCSI volumes, it is recommended to configure
+multipathing.
 
 //*TODO:* Add, or at least mention, the Nova eDPM side for iSCSI.
```

Comment:
Can we remove the "TODO" line?

Reply:
@Akrog Does "these services" refer to Nova services (Compute)? So it would be, "Compute (nova) services are automatically started on data plane nodes."

Note that we refer to "EDPM" as "data plane" downstream.

Reply:
No, these services refer to `iscsid`, `multipathd`, and the nvme-of kernel modules.